packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js

```diff
@@ -552,11 +614,28 @@ class EnumerationWidget extends React.Component {
 		}));
 	}
 
+	renderImportFile() {
+		return (
+			this.allowImport &&
+			<form
+				ref={(element) => { this.formInput = element; }}
+			>
+				<input
+					type="file"
+					ref={(element) => { this.inputImport = element; }}
+					onChange={(event) => { this.importFile(event); }}
+					className={classNames('hidden')}
+				/>
+			</form>
+		);
+	}
+
 	render() {
 		const items = this.searchItems(this.state.searchCriteria);
 		const stateToShow = { ...this.state, items };
 		return (
 			<div>
+				{ this.renderImportFile() }
```

Comment:
What about `{ this.allowImport && this.renderImportFile() }` and remove `this.allowImport &&` in `renderImportFile()` ?